tests/functional/tests.py

```diff
@@ -114,16 +172,12 @@ def test_make_bucket(client):
         # Remove bucket
         client.remove_bucket(bucket_name)
     except Exception as err:
-        log_output['message'] = err
-        log_output['error'] = traceback.format_exc()
-        log_output['status'] = FAIL
-        log_output['duration'] = duration = round(time.time() - start_time, 2)
-        print_json(log_output)
+        print(log_output.json_report_failure(err))
```

Comment:
Good change,

But I still want to to recommend that you only throw the exception here, the main function which is calling all test functions will know that an error occurred and will print it. I think this style is better than `print() > exit()` in each test function.